META.json

```diff
@@ -2,15 +2,15 @@
    "name": "citus",
    "abstract": "Scalable PostgreSQL for real-time workloads",
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "9.1.0",
+   "version": "9.1.1",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
          "file": "src/backend/distributed/sql/citus--9.0-2--9.1-1.sql",
```

Comment:
A minor reminder: This field not really makes sense. We may want to update it with a more sensible placeholder.